src/main/kotlin/app/batch/ConsoleWriter.kt

```diff
@@ -0,0 +1,15 @@
+package app.batch
+
+import org.springframework.batch.item.ItemWriter
+import org.springframework.context.annotation.Profile
+import org.springframework.stereotype.Component
+
+@Component
+@Profile("outputToConsole")
+class ConsoleWriter : ItemWriter<String> {
+    override fun write(items: MutableList<out String>) {
+        items.forEach {
+            println(it)
```

Comment:
Kotlin is mental. What's "it"??? Haha.

Reply:
I'll show you in a demo later :-)
'it' is the object under reference in the '{...}' block :-)